packages/block-library/src/quote/index.js

```diff
@@ -197,11 +197,18 @@ export const settings = {
 		);
 	},
 
-	merge( attributes, attributesToMerge ) {
+	merge( attributes, { value, citation } ) {
+		if ( ! value || value === '<p></p>' ) {
```

Comment:
This is interesting to me as well that it be required to test for. I understand it's a consequence of the multiline behavior, but (correct me if I'm wrong) it's still effectively an emptiness check, and as such it seems strange we'd need to fork logic for concatenating the value of an "empty" field. And also strange that, if it is a test of emptiness, that it not be something we could use `RichText.isEmpty` for.

Alternatively, should we account for this from RichText itself, never allowing `'<p></p>'` to become the value, but instead normalizing to--I assume--the empty string? This feels awfully reminiscent of inconsistencies we had ages ago with the `RichText` array vs. `undefined` values (https://github.com/WordPress/gutenberg/pull/5091#issuecomment-385986656).